.travis.yml

```diff
@@ -1,14 +1,24 @@
 language: python
-
-python:
-  - 2.6
-  - 2.7
-  - 3.3
-  - 3.4
-  - 3.5
-  - 3.6
-  - 3.7-dev
-  - pypy
+matrix:
+  include:
+    - python: 2.7
```

Comment:
If we are dropping 2.6, which is ok, we should update the documentation and and build files appropriately.

We already dropped 2.6 on the other APIs.

Reply:
I think I removed everything to do with 2.6; there's still a comment in `tests/decoder_test.py` but changing the code there didn't seem necessary.